python/uperf/uperf_schema.py

```diff
@@ -79,6 +79,13 @@ class ProfileFlowOpConnection(ProfileFlowOpCommon):
             "description": "The remote host that we need to connect or accept connection from"
         }
     )
+    port: int = field(
+        default=20000,
+        metadata={
+            "name": "port",
+            "description": "The port of the remote server"
```

Comment:
Nit: You use "remote host" in the description above, and "remote server" here. Let's be consistent in the terms used.